### DISCUSSION TITLE
How to set prefix to semantic commit

### DISCUSSION THREAD

**Discussion by amimas** [renovatebot/renovate#10482] [Request Help]
In my self-hosted renovate bot, I am using the `config:base` preset. It opens PRs that look like this:

```
Pin Dependencies
chore(deps): Update xxxx to yyyy
chore(ci): Update xxxx to yyyy
```

I want to add `[Renovate Bot]` prefix to the commit messages and PR titles. I used `commitMessagePrefix: '[Renovate Bot]'`. But, this way I loose the semantic commit types (i.e. `chore(deps)`, `chore(ci)`, etc.). I tried using `semanticCommitType: '[Renovate Bot] chore'` instead of the `commitMessagePrefix` config. But that didn't seem to work; It was opening PRs the same as as shown above.

What can I do so that the PR titles and commits are in this format?

```
[Renovate Bot] Pin Dependencies
[Renovate Bot] chore(deps): Update xxxx to yyyy
[Renovate Bot] chore(ci): Update xxxx to yyyy
```

**Comment by rarkins** [comment]
You might need to play around with `semanticCommitType`. e.g. try  adding `":semanticCommitTypeAll([Renovate Bot] chore"` and checking the result.
